tests/unit/logging/test_profile.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+import pytest
+
+from jina.logging import default_logger
+from jina.logging.profile import TimeDict, profiling
+
+
+@pytest.fixture
+def default_logger_propagate():
+    default_logger.logger.propagate = True
+    yield
+    default_logger.logger.propagate = False
+
+
+def test_logging_profile_profiling(caplog, default_logger_propagate):
+    @profiling
+    def foo():
+        print(1)
```

Comment:
we could allocate some memory here or sleep for some time to showcase how it is used and how the memory and some time is returned.

Tests tend to be good way to clarify behavior

Reply:
Sure! I add some changes. Not sure if this is ok